src/OpenConext/ProfileBundle/Resources/views/MyConnections/available-connections.html.twig

```diff
@@ -0,0 +1,39 @@
+{% set availableConnections = {
```

Comment:
Was this deliberate? If this was not for testing purposes  the availableConnections is empty check below could be removed else the set should be removed.